### PR TITLE
Add IOTA SDK to Next

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* deleted iota.rs, iota.js and wallet.rs submodule entries
* added iota-sdk submodule
* added docs layout component to next section
* removed iota.rs, iota.js and wallet.rs from docs layout and added iota-sdk
* removed iota.rs, iota.js and wallet.rs from develop sidebar and added iota-sdk

## Links to any relevant issues

fixes https://github.com/iotaledger/iota-wiki/issues/957

## Type of change

- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/next/components/DocsLayout/CoreLibrariesSection.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+        <h3 className='nodes__header'>IOTA SDK</h3>
+        <ul className='nodes__features'>
+          <li className='nodes__feature'>
+            <Link to='/iota-sdk/how_tos/accounts_and_addresses/create_address'>
+              Create an address
+            </Link>
+          </li>
+          <li className='nodes__feature'>
+            <Link to='/iota-sdk/simple_transaction/simple_transaction'>
```
Comment (Dr-Electron) [inline src/next/components/DocsLayout/CoreLibrariesSection.tsx:54]:
```suggestion
            <Link to='/iota-sdk/how_tos/simple_transaction'>
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — COMMENTED** [review]
I guess you should update https://github.com/iotaledger/iota-wiki/blob/addSDKToNextSection/next/develop/explanations/what-is-shimmer/clients.md too? 
or do you want to do that in a separate PR?

**Comment by lucas-tortora** [comment]
> I guess you should update https://github.com/iotaledger/iota-wiki/blob/addSDKToNextSection/next/develop/explanations/what-is-shimmer/clients.md too? or do you want to do that in a separate PR?

I think it'd be better once we merge the docs PRs in the SDK. So far, there aren't many links we can add, so we'll need to go back once we have everything

**Review by Dr-Electron — APPROVED** [review]
Ok, sounds good to me

**Pull request merged**